### DISCUSSION TITLE
I can't put mods on lazer!

### DISCUSSION THREAD

**Discussion by Haeybu** [ppy/osu#23017] [Q&A]
hello i have problems  my osu lazer can't have mod he say "could not load ruleset "..." please check for an update from the developper

Can you help me ?

**Comment by bdach** [comment]
Do you have custom rulesets installed?

**Reply by Haeybu** [reply]
i think yes  it's swims and hitakori

**Reply by bdach** [reply]
Then please update them (if updates are available), or remove them for now until they are updated to work properly with the latest release.

**Reply by Haeybu** [reply]
ok or i can use a old version of lazer ?

**Reply by bdach** [reply]
If you insist, yes.

**Reply by Haeybu** [reply]
ok i waiting for update :) thank you
